packages/config-plugins/src/ios/__tests__/ProvisioningProfile-test.ts

```diff
@@ -13,6 +13,45 @@ describe('ProvisioningProfile module', () => {
     const projectRoot = '/testproject';
     const pbxProjPath = 'ios/testproject.xcodeproj/project.pbxproj';
 
+    afterEach(() => {
+      vol.reset();
+    });
+
+    it('configures the project.pbxproj file for multitarget projects', () => {
+      vol.fromJSON(
+        {
+          [pbxProjPath]: originalFs.readFileSync(
+            path.join(__dirname, 'fixtures/project-multitarget.pbxproj'),
+            'utf-8'
+          ),
+        },
+        projectRoot
+      );
+      setProvisioningProfileForPbxproj(projectRoot, {
```

Comment:
you are not testing it as multitarget project  because you did not specify which target  should be updated